staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/admission.go

```diff
@@ -275,8 +275,19 @@ func (a *GenericAdmissionWebhook) callHook(ctx context.Context, h *v1alpha1.Exte
 		return fmt.Errorf("admission webhook %q denied the request without explanation", h.Name)
```

Comment:
Can you add a test? For example this line should also be returning a status error.

Reply:
Yeah, maybe move line 270-275 to the `toStatusErr` function as well.